Importing Tracking plan (#5384)
segmentio/segment-docs: src/protocols/faq.md

Unplanned property omission is only supported for cloud-mode destinations. Unplanned properties will not be omitted when they're sent to device-mode destinations.

> ### How to import events from the Source Schema to a Tracking Plan

Reviewer: ```suggestion
### Can I import events from my Source Schema into a Tracking Plan?
```

> You can import events into the tracking from the source schema when you initially create your tracking plan. Also, you can manually add 
> these events by clicking the the (+) next to the event in your source schema page once the tracking plan has been connected.  

Reviewer: ```suggestion
When you initially create your Tracking Plan, you can import events into it from a Source Schema. Manually add these events by clicking the the (+) next to the event in your Source Schema page after connecting your Tracking Plan. .  
```
